src/Components/Display-card/DisplayCard.jsx

```diff
@@ -0,0 +1,31 @@
+import "../../css/main.css";
+import "./displaycard.css";
+
+export const DisplayCard = ({color}) => {
+    return(
+        <>
+        <div className="display-card flex flex-column pd-1" style={{backgroundColor: color}}>
```

Comment:
Suggestion: Avoid Inline styling.

Reply:
Yes even I tried to not use inline styling but the scenario here is I have a common display note component and I am using it in various places and I need to only change the background color on the display note , so I am passing the color as a prop to the display note component. So to use this prop value as the background color I had no other option but to use inline styling. If I can do it in some other way please let me know.